### PR TITLE
Adding dependency information to README

### PR DESCRIPTION
**Pull request by Izaakwltn**
This adds information about two current dependencies, `FSET` and `MISC-EXTENSIONS`, to the README.

### CODE REVIEW

**Review by jbouwman — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -116,7 +116,9 @@ Coalton is currently used in production to build [quantum computing software](ht
 
 **Install**: Clone this repository into a place your Lisp can see (e.g., `~/quicklisp/local-projects/`).
 
-**Use**: Either run `(ql:quickload :coalton)`, or add `#:coalton` to your ASD's `:depends-on` list. Quicklisp will automatically download all of Coalton's dependencies.
+**Add dependencies** While Quicklisp will automatically ensure almost all of Coalton's dependencies, you will need to manually clone [FSET](https://github.com/slburson/fset/) and [MISC-EXTENSIONS](https://github.com/slburson/misc-extensions/) and add them to your asdf/quicklisp path (e.g., `~/quicklisp/local-projects/`).
```
Comment (stylewarning) [inline README.md:119]: Capitalization: ASDF and Quicklisp

**Review by stylewarning — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Izaakwltn) [reply README.md]: Fixed

**Comment by Izaakwltn** [comment]
Should this also mention the dependency on GNU-MPFR in order to run tests?

**Comment by stylewarning** [comment]
> Should this also mention the dependency on GNU-MPFR in order to run tests?

probably

**Review by stylewarning — APPROVED** [review]
Solid!

**Pull request merged**